File: WindowsApplication1/WindowsApplication1/Form2.vb
Public Class Form2

    Private Sub txt_first_Enter(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles txt_first.Enter
        txt_first.Text = "welcome"
        txt_second.Text = "By"
    End Sub

    Private Sub txt_first_Leave(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles txt_first.Leave
        txt_first.Text = "By"
        txt_second.Text = "welcome"
    End Sub
End Class

File: WindowsFormsApplication1/WindowsFormsApplication1/Form2.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace WindowsFormsApplication1
{
    public partial class Form2 : Form
    {
        public Form2()
        {
            InitializeComponent();
        }

        private void txt_first_Enter(object sender, EventArgs e)
        {
            txt_first.Text = "Welcome";
            txt_second.Text = "By";
        }

        private void txt_first_Leave(object sender, EventArgs e)
        {
            txt_first.Text = "By";
            txt_second.Text = "Welcome";
        }

    }
}
